ddtrace/context.py

```diff
@@ -29,6 +29,21 @@ def __init__(self):
         self._finished_spans = 0
         self._current_span = None
         self._lock = threading.Lock()
+        self._base_parent_span_id = None
+        self._base_parent_trace_id = None
+
+    def set_base_parent_span_ids(self, trace_id, span_id):
```

Comment:
also helpers.run_in_executor is broken for ProcessPoolExecutors, I'd imagine it should least have a warning